src/wallet/asyncrpcoperation_saplingmigration.cpp

```diff
@@ -126,21 +140,30 @@ bool AsyncRPCOperation_saplingmigration::main_impl() {
         builder.AddSaplingOutput(ovkForShieldingFromTaddr(seed), migrationDestAddress, amountToSend - FEE);
         CTransaction tx = builder.Build().GetTxOrThrow();
         if (isCancelled()) {
+            LogPrint("zrpcunsafe", "%s: Canceled. Stopping.\n");
             break;
         }
         pwalletMain->AddPendingSaplingMigrationTx(tx);
+        LogPrint("zrpcunsafe", "%s: Added pending migration transaction with txid=%s\n", getId(), tx.GetHash().ToString());
         ++numTxCreated;
-        amountMigrated += amountToSend;
+        amountMigrated += amountToSend - FEE;
```

Comment:
This is to prevent confusion between this value and the `amount_migrated` returned by `z_getmigrationstatus`. It probably should have excluded the FEE originally.